Parameters.py

```diff
@@ -1,41 +1,42 @@
+"""Command line arguments parser configuration."""
 import argparse  # default python library for command line argument parsing
 
-parser = argparse.ArgumentParser(description='Train DNNs on model car data.',
+PARSER = argparse.ArgumentParser(description='Train DNNs on model car data.',
```

Comment:
PARSER isn't a constant since we call it's helper functions which modify it's attributes several times over this file should be changed to "parser"